static/js/pages/instances/InstanceList.tsx

```diff
@@ -263,7 +263,35 @@ const InstanceList: FC = () => {
                 isLoading ? (
                   <Loader text="Loading instances..." />
                 ) : (
-                  "No data to display"
+                  <Row className="empty-state-message">
```

Comment:
In the snapshot table, the column names are not shown with the empty state. Can make it the same here? Not sure why this differs though.
Or at least at some spacing to the above element, the empty states is a bit cramped below the table headers atm.

Reply:
Instance list:

![image](https://user-images.githubusercontent.com/56583786/212854517-a3010e7e-960e-4038-ba7f-45f6f2b2740a.png)

Snapshot list:

![image](https://user-images.githubusercontent.com/56583786/212854713-b7b40f49-0ad8-413a-8286-f59c3133a060.png)

I don't see any difference and the column headers are showing in both cases. I will add some spacing to both of them, so that hopefully they look less cramped.

Reply:
I didn't get the table headers in empty snapshots yesterday, confused why they appear today.

I still think it would look better with 3rem margin above the empty states.

Reply:
The old version of the snapshot empty state, which was not using the `emptyStateMsg` prop, used to be without the column headers, because it was conditionally rendered instead of the table itself. But that was the first change that I applied - even before the one for instances - as reflected by the commit order, so probably you were running a cached version of the app or wrong branch.

I have changed the margin to 3rem, but I will also check Ant's link now and commit again soon.

Reply:
From the example in vanilla I guess we should use a wrapper with the `p-strip` class to get the 3rem spacing.